### PR TITLE
feat: #150 add an accordion component to the  about project page (150)

### PR DESCRIPTION
**Pull request by MaratReactJs**
Added an accordion component to the  about project page

### CODE REVIEW

File: mw-webapp/src/logic/aboutProjectPage/AboutProjectPage.module.scss
Comment (SergioAJS) [inline mw-webapp/src/logic/aboutProjectPage/AboutProjectPage.module.scss:1]: Please use constants for margin. Check project style files for eample

**Review by SergioAJS — CHANGES_REQUESTED** [review]

File: mw-webapp/src/logic/aboutProjectPage/AboutProjectPage.module.scss
Comment (MaratReactJs) [reply mw-webapp/src/logic/aboutProjectPage/AboutProjectPage.module.scss:1]: Thanks, added constant for margin and used it

**Review by SergioAJS — APPROVED** [review]

**Review by tritonJS826 — APPROVED** [review]
Cool! Welcome aboard!

**Review by tritonJS826 — APPROVED** [review]

File: mw-webapp/src/logic/aboutProjectPage/AboutProjectPage.tsx [hunk 1]
```diff
@@ -1,13 +1,42 @@
...
-    <Title
-      level={HeadingLevel.h2}
-      text="About project page"
-    />
+    <>
+      <Title
+        level={HeadingLevel.h2}
+        text="About project page"
+      />
+      <>
```
[the remark this answers is not in the document]
Comment (tritonJS826) [reply mw-webapp/src/logic/aboutProjectPage/AboutProjectPage.tsx]: Please comment each comment) something like "removed unneeded react.fragment"

[the remark this answers is not in the document]
Comment (MaratReactJs) [reply mw-webapp/src/logic/aboutProjectPage/AboutProjectPage.tsx]: Sorry I didn’t see it here, I just watched it in the video.
The extra react fragment has been removed, thanks

**Review by Ekaterina1994 — APPROVED** [review]

**Pull request merged**